src/lib/Carousel/Carousel.svelte

```diff
@@ -108,9 +108,22 @@
   }
 
   onMount(() => {
+    const carouselContainer = document.querySelector('.carousel-container');
```

Comment:
We shouldn't do this. instead use bind:this in <div class="carousel-container" >